core/src/main/kotlin/agent/Dto.kt

```diff
@@ -61,9 +62,10 @@ fun AgentInfo.toDto(agentManager: AgentManager, isList: Boolean = false): AgentI
         ipAddress = ipAddress,
         activePluginsCount = this@toDto.plugins.activePluginsCount(),
         sessionIdHeaderName = sessionIdHeaderName,
-        plugins = this@toDto.plugins.mapToDto().toSet(),
+        plugins = plugins.values.ofAgent(this@toDto).mapToDto().toSet(),
```

Comment:
This is ugly. We need to rewrite the method.
Also, this "isList" flag is totally useless. Remove it.
Finally, move it to Extensions.kt.
Here's an example
```kotlin
fun AgentInfo.toDto(agentManager: AgentManager) = AgentInfoDto(
    id = id,
    serviceGroup = serviceGroup,
    instanceIds = agentManager.instanceIds(id),
    name = name,
    description = description,
    environment = environment,
    status = status,
    buildVersion = buildVersion,
    adminUrl = adminUrl,
    ipAddress = ipAddress,
    activePluginsCount = plugins.activePluginsCount(),
    sessionIdHeaderName = sessionIdHeaderName,
    plugins = agentManager.plugins.ofAgent(this).mapToDto().toSet(),
    packagesPrefixes = agentManager.adminData(id).packagesPrefixes,
    agentType = agentType.notation,
    agentVersion = agentVersion
)
```